Makefile

```diff
@@ -31,6 +31,12 @@ mexport.so: main.go go.mod
 	$(GOCC) build $(GOFLAGS) -buildmode=plugin -o "$@" "$<"
 	chmod +x "$@"
 
+gen-out:
+	mkdir -p out
+	docker create --name extract kubo-mexport
+	docker cp extract:/usr/local/bin/ipfs ./out/
```

Comment:
This is still a single binary and will thus not extract the plugin:
```
Successfully tagged kubo-mexport:latest
leo@leo-thinkpad:~/tmp/ipfs-metric-exporter$ make gen-out
mkdir -p out
docker create --name extract kubo-mexport
db20b6b310959725af20edf86ca57ee77408f9638a3abddfbbfd4d221d7a8465
docker cp extract:/usr/local/bin/ipfs ./out/
docker rm extract
extract
leo@leo-thinkpad:~/tmp/ipfs-metric-exporter$ l out/
total 78M
drwxrwxr-x 2 leo leo 4.0K Jan 30 14:11 .
drwxrwxr-x 7 leo leo 4.0K Jan 30 14:11 ..
-rwxr-xr-x 1 leo leo  78M Dec 22 14:54 ipfs
```

Reply:
Ohh I misunderstood what you said earlier. Got you. Working on it...

Reply:
Ok, we now get
```
# ls -l out
total 139632
-rwxr-xr-x 1 root root 81029040 Jan 12 16:46 ipfs
-rwxr-xr-x 1 root root 61950808 Jan 30 09:36 mexport-v0.17.0-docker.so
```